### DISCUSSION TITLE
Add new table to Db

### DISCUSSION THREAD

**Discussion by applePieFace** [fretje/heroplate#16] [Q&A]
How do i add a new table to the db, which file do i add the table definition to?

**Comment by fretje** [comment]
All database configuration is now in src/Api/Api.Infrastructure/Persistence/Configuration/Heroplate.cs

For example this is the configuration for the Product entity:
https://github.com/fretje/heroplate/blob/bc379f3e981a1c7851e9c12fd5fde40ca73825ee/src/Api/Api.Infrastructure/Persistence/Configuration/Heroplate.cs#L24-L32

But you can put them anywhere in the infrastructure project.

They will automatically get picked up by this:

https://github.com/fretje/heroplate/blob/bc379f3e981a1c7851e9c12fd5fde40ca73825ee/src/Api/Api.Infrastructure/Persistence/Context/BaseDbContext.cs#L39
